assets/css/tailwind.css

```diff
@@ -0,0 +1,20 @@
+@import 'tailwindcss/base';
+@import 'tailwindcss/components';
+@import 'tailwindcss/utilities';
+
+.newButton{
```

Comment:
As mentioned, please use Tailwinds Arbitrary syntax ☺️ 
If that is not possible at all, Vue's scoped styles are also fine, but then use `@apply` where possible